### ISSUE TITLE
Data source

### ISSUE THREAD

**Issue by paul90** [fedwiki/wiki-plugin-pushpin#5]
Currently the data source for this plugin is hardcoded.

It would be nice if it searched for data sources within the lineup, would go very nicely with the datscript plugin...

**Comment by WardCunningham** [comment]
Yes. Yes. Let's generalize the pushpin map and test it with Dat.

![image](https://cloud.githubusercontent.com/assets/12127/8377961/a6db7dce-1bca-11e5-9401-e38904b13be2.png)
&nbsp; _A pushpin map. See [about page](http://asia.fed.wiki.org/view/about-pushpin-plugin/view/more-about-pushpin-plugin) for animation effect._

The pushpin approach is tolerant of strongly clustered data, such as the earthquake data from the Dat example. The underlying technology (d3.js) is also good at rendering transitions. It is my own untested hypothesis that the two could combine to animate time-varying geolocated data in a way that would be very intuitive for the viewer. Imagine scrubbing over a month of earthquake data and watching the progression of aftershocks.

This is a related d3 experiment that does animate in conjunction with force layout.
http://bl.ocks.org/wardcunningham/5861122

So pushpin should look for geocoordinates. These could have event times associated with them, perhaps start and end times. They could have size and color too. There would need to be a number of transformation steps to get from Dat to Pushpin, but these steps are easily within the computational power of client-side wiki, almost boringly simple, but not easy to modularize into reusable plugins.

Let's launch a project around this one case, Eartquake => Dat => Wiki => Pushpin, and make something suggestive of how this sort of data wrangling could be composed interactively.

With this issue, the project is launched. https://github.com/fedwiki/wiki-plugin-pushpin/issues/5
